chart-values/elasticsearch_values.yaml

```diff
@@ -111,7 +111,7 @@ client:
     minAvailable: 1
     # maxUnavailable: 1
   ingress:
-    enabled: true
+    enabled: false
```

Comment:
you can set to false and delete the other values, just keep the enabled: false

Reply:
@cpanato removed the non needed section.